importer/importer.go

```diff
@@ -132,14 +136,28 @@ func (im *Importer) executeSQL(server servers.ClickhouseServer, sql string) erro
 }
 
 func (im *Importer) createTable(server servers.ClickhouseServer, user apikeys.APIKeyDetails, table string) error {
+	engine := "MergeTree"
+	clusterStmt := ""
+	if cluster := user.GetDBCluster(); cluster != "" {
+		shard, replica := user.GetDBShard(), user.GetDBReplica()
+		if shard == "" || replica == "" {
+			return errors.New("distributed tables require shard and replica names")
+		}
+		engine = fmt.Sprintf(
+			"ReplicatedMergeTree('/clickhouse/%s/tables/%s/{database}/{table}', '%s')",
```

Comment:
This should be

```suggestion
			"ReplicatedMergeTree('/clickhouse/{%s}/tables/{%s}/{database}/{table}', '{%s}')",
```

We're not defining the name of the shard/replica. We're defining the name of the _macro_ which has the shard/replica.